worlds/sm64ex/docs/en_Super Mario 64 EX Advance Setup.md

```diff
@@ -0,0 +1,71 @@
+# Super Mario 64 EX Advanced Setup Guide
+
+## Required Software
+
+- Super Mario 64 US Rom (Japanese may work also. Europe and Shindou not supported)
+- Either of [sm64pclauncher](https://github.com/N00byKing/sm64pclauncher/releases) or
+- Cloning and building [sm64ex](https://github.com/N00byKing/sm64ex) manually.
+
+NOTE: The above linked sm64pclauncher is a special version designed to work with the Archipelago build of sm64ex.
+You can use other sm64-port based builds with it, but you can't use a different launcher with the Archipelago build of sm64ex.
+
+## Building with Textures, Models and Patch Files
+
+# Installation via sm64pclauncher (For Windows/Linux)
+First, install [MSYS](https://www.msys2.org/) as described on the page. DO NOT INSTALL INTO A FOLDER PATH WITH SPACES.
+Do all steps up to including step 6.
+Best use default install directory.
+Then follow the steps below
+
+1. Go to the page linked for sm64pclauncher, and press on the topmost entry
+3. Scroll down, and download the zip file
+4. Unpack the zip file in an empty folder
+5. Run the Launcher and press build.
+6. Set the location where you installed MSYS when prompted. Check the "Install Dependencies" Checkbox
+7. Set the Repo link to `https://github.com/N00byKing/sm64ex` and the Branch to `archipelago` (Top two boxes). You can choose the folder (Secound Box) at will, as long as it does not exist yet
+8. Set the location for the Models Pack
+9. Set the location for the Texture Pack
+10. Point the Launcher to your Super Mario 64 US/JP Rom, and set the Region correspondingly
+11. Copy the .patch file/s to the enachancements folder which is located in e.g. C:/sm64pclauncher/apmario/enhancements

```

Comment:
Add a note that outside patches may break the game. Only the ones supplied with the repo are tested

Reply:
fixed